src/objects/core/zcl_abapgit_serialize.clas.abap

```diff
@@ -224,7 +226,14 @@ CLASS zcl_abapgit_serialize IMPLEMENTATION.
       iv_language         = io_dot_abapgit->get_main_language( )
       ii_log              = ii_log
       iv_force_sequential = lv_force ).
-    APPEND LINES OF lt_found TO ct_files.
+* transfer one by one and delete from original table as APPEND LINES OF
+* leads to doubling memory usage during execution of statement which
+* could result in SYSTEM_NO_ROLL dump
+    LOOP AT lt_found ASSIGNING <ls_found>.
+      lv_index = sy-tabix.
+      APPEND <ls_found> TO ct_files.
+      DELETE lt_found INDEX lv_index.
```

Comment:
Storing and using the index to delete from table is pointless as the sy-tabix will always be 1 if you are deleting from the table.

Reply:
You're right. Done.